bob-backend/src/core.rs

```diff
@@ -445,38 +450,57 @@ impl Backend {
         self.inner.free_least_used_resources().await
     }
 
-    pub async fn delete(&self, key: BobKey) -> Result<u64, Error> {
+    pub async fn delete(
+        &self,
+        key: BobKey,
+        options: BobOptions,
+        force_delete: bool,
```

Comment:
`force_delete` should be removed. For normal deletion it is always true, for aliens it is estimated from `options` according to the `force_alien_nodes`